#!/usr/bin/env python
PACKAGE = "rt_gene"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("model_size", double_t, 0, "Scale of the 3D model of the face used for head pose estimation", 30.0, 1.0, 100.0)
gen.add("head_pitch", double_t, 0, "Alter the head pose by the head_pitch", 0.0, -10.0, 10.0)

exit(gen.generate(PACKAGE, "rt_gene", "ModelSize"))
